internal/provider/resource_ingestionkey.go

```
@@ -0,0 +1,260 @@
+
+	tflog.Debug(ctx, "Sending CreateIngestionKeyRequest to SDK", map[string]any{"name": nameStr, "type": typeStr, "remote_config": remoteConfig, "tags": tags})
+	apiResponse, err := r.client.CreateIngestionKey(ctx, createReq)
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Error Creating Ingestion Key",
+			fmt.Sprintf("Could not create Ingestion Key: %s", err.Error()),
+		)
+		return
+	}
+
+	tflog.Debug(ctx, fmt.Sprintf("Ingestion Key created %s", apiResponse.Name))
```

golangci-lint (1.64.8)

printf: fmt.Sprintf format %s has arg apiResponse.Name of wrong type *string

(govet)